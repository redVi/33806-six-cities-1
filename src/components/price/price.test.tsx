import React from 'react';
import renderer from 'react-test-renderer';
import Price from '@/components/price/price';

describe(`Price`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(<Price price={128} text="/night" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
