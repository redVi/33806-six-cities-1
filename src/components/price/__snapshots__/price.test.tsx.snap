// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Price renders correctly 1`] = `
<div
  className="place-card__price"
>
  <b
    className="place-card__price-value"
  >
    â‚¬
    128
  </b>
  <span
    className="place-card__price-text"
  >
    /night
  </span>
</div>
`;
